crates/driver/src/domain/competition/solution/trade.rs

```diff
@@ -97,6 +95,11 @@ impl Fulfillment {
         }
     }
 
+    /// Returns the raw form of the fee
```

Comment:
I think a comment is needed here, but in its current form it doesn't really help me much given the function name. What do we mean by raw fee? Is this another type of fee we need to define in our glossary? How is it different to just `fee` ?

Reply:
Added a different comment, hopefully better.